lib/service/service.go

```diff
@@ -2065,6 +2067,26 @@ func (process *TeleportProcess) initSSH() error {
 	return nil
 }
 
+func compareVersions(c *Connector) error {
+	sver, err := c.Client.GetVersion(context.TODO())
+	if err != nil {
+		return trace.Wrap(err)
+	}
+	serverVersion, err := semver.NewVersion(sver)
+	if err != nil {
+		return trace.Wrap(err)
+	}
+	teleportVersion, err := semver.NewVersion(teleport.Version)
+	if err != nil {
+		return trace.Wrap(err)
+	}
+	if math.Abs(float64(serverVersion.Major-teleportVersion.Major)) > 2 {
```

Comment:
Wonder why we have `>2` major version condition here 
For example the [Teleport versioning](https://github.com/gravitational/teleport/blob/master/rfd/0012-teleport-versioning.md#compatibility) RFD mentions only about `vN-1.*.*` supported compatibility

Reply:
Ah I was going by support window below it